httomo/runner/task_runner.py

```diff
@@ -340,7 +344,7 @@ def determine_max_slices(self, section: Section, slicing_dim: int):
 
             output_dims = m.calculate_output_dims(non_slice_dims_shape)
             (slices_estimated, available_memory) = m.calculate_max_slices(
-                self.source.dtype,
+                np.dtype('float32'), # self.source.dtype, # see the NOTE about the dtype above
```

Comment:
This is a nitpick, but this looks a bit funny due to the double `#` (I'm assuming the old code was commented out?).

Also, being a bit more explicit about something that's hardcoded might be useful.

Could you maybe define a variable:
```python
SOURCE_DTYPE = np.dtype('float32')
```
alongside the main comment added at line 335, and then change this line to something like the following, to remove the double `#`:
```python
SOURCE_DTYPE, # see the NOTE about the dtype above
```